src/db.ls

```diff
@@ -67,16 +67,58 @@
     db.DB = {}
     minimatch = require \minimatch
     try
-      db.DB = JSON.parse do
-        require \fs .readFileSync "#dataDir/dump.json" \utf8
-      console.log "==> Restored previous session from JSON file"
+      db.DB =
+        save_timestamps: {}
+        timestamps: {}
+      if fs.existsSync "#dataDir/dump/"
+        fs.readdirSync("#dataDir/dump/").forEach (f) ->
+          key = f.split(".")[0]
+          type = key.split("-")[0]
+          id = key.split("-")[1]
+          db.DB.timestamps["timestamp-#id"] = 0
```

Comment:
To avoid pollution of timestamps, I'd recommend setting here _if_ any file content is read.

```ls
v =
    switch type
    | \snapshot => fs.read-file-sync "#dataDir/dump/#key.txt", \utf8
    | \audit => fs.read-file-sync "#dataDir/dump/#key.txt", \utf8 .split "\n"
    | _ => void
db.DB[key] = v if v?
```